specification/sections/01-flag-evaluation.md

```diff
@@ -423,19 +425,23 @@ stateDiagram-v2
     READY --> NOT_READY:shutdown()
     STALE --> NOT_READY:shutdown()
     ERROR --> NOT_READY:shutdown()
-    READY --> CONTEXT_PENDING:::optional:setContext()
-    CONTEXT_PENDING --> READY:*
+    READY --> RECONCILING:::client:setContext()
+    RECONCILING:::client --> READY:*
 
-    classDef optional fill:#999
+    classDef client fill:#555
 ```
 
 \* transitions occurring when associated events are spontaneously emitted from the provider
 
-Note that only SDKs implementing the [static context (client-side) paradigm](../glossary.md#static-context-paradigm) define `CONTEXT_PENDING` to facilitate [context reconciliation](./02-providers.md#26-provider-context-reconciliation).
+<span style="color:#555">█</span> only defined in static-context (client-side) paradigm
```

Comment:
Clever 😄

Reply:
I thought so. :sunglasses:

Reply:
It's a bit hard to see, though. Can you try a different color?

![image](https://github.com/open-feature/spec/assets/682996/b6f2c7dd-c6d8-4692-a64c-0d626bf5ab9b)

Reply:
I'll move it back to 888

Reply:
I was also just trying out any gray colors😅
I like what you did with the comment!

Reply:
I've used 888, which is in the middle and looks OK in both IMO